docs/CHANGELOG.rst

```diff
@@ -15,6 +15,11 @@ Changed
 - Add ``username`` to ``current_user_permissions`` field of objects on API
 
 
+Fixed
+-----
+- Support worflows as inputs to Python processes
```

Comment:
If change log entry starts with the "Support", then it is not a fix. Move it under "Added" or rephrase it to "Fix workflows to work with..."

Reply:
Done.